From 94d10dd1563a718a0034b47a94e121216b4a145c Mon Sep 17 00:00:00 2001
From: Stefan Roese <sr@denx.de>
Date: Wed, 15 Aug 2018 16:35:43 +0200
Subject: [PATCH] mips: mt7628a.dtsi: Add SPI clock-frequency property

This patch adds the clock-frequency property to the SPI controller
DT node. It will be used by the SPI driver to calculate the baud rate.

Signed-off-by: Stefan Roese <sr@denx.de>
Cc: Daniel Schwierzeck <daniel.schwierzeck@gmail.com>
---
 arch/mips/dts/mt7628a.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/mips/dts/mt7628a.dtsi b/arch/mips/dts/mt7628a.dtsi
index d525f019fe..c14259b170 100644
--- a/arch/mips/dts/mt7628a.dtsi
+++ b/arch/mips/dts/mt7628a.dtsi
@@ -76,6 +76,8 @@
 			reg = <0xb00 0x40>;
 			#address-cells = <1>;
 			#size-cells = <0>;
+
+			clock-frequency = <200000000>;
 		};
 
 		uart0: uartlite@c00 {
